### ISSUE TITLE
CollectionView reloadSections doesn't remove any text or images

### ISSUE THREAD

**Issue by Ivan-Cantarino** [dzenbot/DZNEmptyDataSet#383]
Hello.
So I was testing this library, which everyone talks so good about and I've ran into an issue with it.
In my app, after fetching some data I call `self.collectionView?.reloadSections((NSIndexSet(index: 0) as IndexSet))` so I can reload the first section in my `UICollectionView` but as I do that, I've noticed the `DZNEmptyDataSet title, image, description` doesn't get removed, but it stays visible, overlapping the cells itself.

If I switch from the `reloadSections(:)` to the good old `reloadData()` it works as expected.

I've looked around and couldn't find any bug report like this one so I think this might need a fix somehow.

Everything else works as expected. 😉

**Comment by Ivan-Cantarino** [comment]
I don’t. Only 1 section 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by Ivan-Cantarino** [comment]
@yangzheng006 @dzenbot actually there's not a solution for it -- I have to call `reloadData()` 😢

**Comment by matthewfx** [comment]
yeah, that one is very frustrating... when one uses RxDataSource it often uses reload sections...

**Comment by flovaguet** [comment]
I had the same issue using tableview.reloadData(with: .automatic) and solve the issue by calling tableview.reloadEmptyDataSet()
